lib/ood_core/job/adapters/kubernetes/helper.rb

```diff
@@ -195,10 +195,14 @@ def service_info_from_json(json_data)
   end
 
   def secret_info_from_json(json_data)
-    raw = json_data.dig(:data, :password)
-    { ood_connection_info: { password: Base64.decode64(raw) } }
-  rescue
-    {}
+    data = json_data.to_h[:data] || {}
+
+    info = data.symbolize_keys.each_with_object({}) do |data_kv, hash|
+      hash[data_kv[0]] = Base64.decode64(data_kv[1])
+    rescue
+      next
+    end
+    { ood_connection_info: info }
```

Comment:
When this returns the `info` will be a hash itself with many password keys and values. Will that be an issue since previously we returned a hash with a single `password` key? 

It looks good, just trying to think if that would cause an issue of a returning hash now having many `password` keys that previously had been used with just the one `password` key.

Reply:
Ok, after looking at the code closer I see this won't be an issue. The method is `private` and being used later with a `deep_merge` into the `pod_hash` which handles all this fine.

Looks good!